funcs.go

```diff
@@ -43,6 +45,6 @@ type FlagEnvHintFunc func(envVars []string, str string) string
 // with the file path details.
 type FlagFileHintFunc func(filePath, str string) string
 
-// InterruptHandlerFunc is used to define the behaviour when a program is interrupted
+// InterruptHandler is used to define the behaviour when a program is interrupted
```

Comment:
The docs should till have the Func in the name

Reply:
Will update this